Misc/NEWS.d/next/Library/2017-12-27-21-55-19.bpo-31639.l3avDJ.rst

```diff
@@ -0,0 +1,2 @@
+http.server now exposes a ThreadedHTTPServer class and uses it when the
+module is invoked to cope with web browsers pre-opening sockets.
```

Comment:
"and uses it when the module is invoked" ?  Doesn't the user have to explicitly use the new class?

Reply:
By "module is invoked" I mean running "python3 -m http.server". In the "python -m http.server" case, the `ThreadedHTTPServer` is used, to fix bpo-31639.

Reply:
So perhaps change "is invoked" to "is run with -m" ?